.github/workflows/integration_test.yml

```diff
@@ -51,17 +56,27 @@ jobs:
       - name: Terraform Apply
         run: |
           echo "${{ vars.TFVARS }}" | base64 --decode | jq  > "infra/tfvars.json"
-          terraform -chdir="infra" apply -auto-approve -var="resource_group_name=ci-${{ github.run_id }}" -var="key_vault_name=ci${{ github.run_id }}" -var-file=tfvars.json
+          terraform -chdir="infra" apply -auto-approve -var="resource_group_name=ci-${{ matrix.run }}${{ github.run_id }}-acme" -var="key_vault_name=ci${{ matrix.run }}${{ github.run_id }}" -var-file=tfvars.json
```

Comment:
Minor: Matrix should be suffix of run id --> "${{ github.run_id }}-${{ matrix.run }}" / "${{ github.run_id }}${{ matrix.run }}"

Reply:
True - thanks